mlir/include/mlir/Dialect/Bufferization/IR/BufferDeallocationOpInterface.h

```diff
@@ -0,0 +1,217 @@
+//===- BufferDeallocationOpInterface.h --------------------------*- C++ -*-===//
```

Comment:
The first line of header files should end with `--*- C++ -*-===//` if there's enough space.